core/src/test/scala/unit/kafka/tools/StorageToolTest.scala

```diff
@@ -354,6 +354,7 @@ Found problem:
         MetadataVersion.LATEST_PRODUCTION,
         Map(TestFeatureVersion.FEATURE_NAME -> featureLevel),
         allFeatures,
+        false,
```

Comment:
Hmm, TEST_VERSION level 2 is not in production and should show an exception when calling `StorageTool.generateFeatureRecord`?